src/adhocracy_core/adhocracy_core/scripts/test_auto_transition_workflow.py

```diff
@@ -0,0 +1,206 @@
+from datetime import datetime
+from pyramid import testing
+from unittest.mock import Mock
+from pytest import fixture
+
+
+class TestAutoTransitionWorkflow:
+
+    def call_fut(self, *args):
+        from .auto_transition_process_workflow import _auto_transition_process_workflow
+        return _auto_transition_process_workflow(*args)
+
+    @fixture
+    def registry(self, registry_with_content):
+        return registry_with_content
+
+    @fixture
+    def mock_catalogs(self, mocker, mock_catalogs):
+        mocker.patch(
+            'adhocracy_core.scripts.auto_transition_process_workflow.'
+            'find_service',
+            return_value = mock_catalogs)
+        return mock_catalogs
+
+    @fixture
+    def mock_transition_to_states(self, registry, mocker):
+        mock = mocker.patch(
+            'adhocracy_core.scripts.auto_transition_process_workflow.'
+            'transition_to_states')
+        return mock
+
+    @fixture
+    def mock_workflow(self, registry, mock_workflow):
+        mock_workflow.get_next_states.return_value = ['evaluate']
+        registry.content.get_workflow = Mock(return_value=mock_workflow)
+        return mock_workflow
+
+    @fixture
+    def mock_now(self, mocker):
+        mock = mocker.patch(
+            'adhocracy_core.scripts.auto_transition_process_workflow.now',
+            return_value = datetime(2016, 1, 5))
+        return mock
+
+    def test_ignore_no_processes(self, context, registry, mocker,
+            mock_catalogs, mock_transition_to_states):
+        self.call_fut(context, registry)
+        assert not mock_transition_to_states.called
+
+    def test_ignore_processes_without_auto_transition(self, context, registry,
+            mock_catalogs, search_result, mock_transition_to_states,
+            mock_sheet):
+        process = testing.DummyResource()
+        context['process'] = process
+        mock_catalogs.search.side_effect = [
+            search_result._replace(elements=[process]),
+            search_result]
+        registry.content.workflows_meta = {
+                'standard': {'auto_transition': False}}
+        mock_sheet.get.return_value = {
+            'workflow': 'standard',
+            'workflow_state': 'participate',
+            'state_data': [
+                {'name': 'participate',
+                 'description': '',
+                 'start_date': datetime(2016, 1, 1),
+                 'end_date': datetime(2016, 1, 2)}
```

Comment:
some statements like creating the process, setup the search mock and setup the workflow meta is repeated in every test. You could move this to a test fixture.